catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/WfsFilterDelegate.java

```diff
@@ -311,10 +311,9 @@ public FilterType and(List<FilterType> filtersToBeAnded) {
 
     @Override
     public FilterType or(List<FilterType> filtersToBeOred) {
-        if (filtersToBeOred.contains(Collections.singleton(null))) {
-            throw new UnsupportedOperationException("Invalid filters found in list of filters.");
+        if (filtersToBeOred.contains(null)) {
```

Comment:
@codice/ogc is anybody familiar with this code? It was a finding because the condition could never evaluate to a true. I believe the desired behavior was to check for a null FilterType. I changed it and updated a test in TestWfsFilterDelegate.java that seemed to be testing that case. Is that correct?